### PR TITLE
Removed duplicate array keys project-wide

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Removed duplicate array keys project-wide

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: tests/unit/suites/libraries/cms/html/JHtmlStringTest.php [hunk 1]
```diff
@@ -139,14 +139,6 @@ public function getTestTruncateData()
 			// Don't return invalid HTML
 			'Plain html over the limit splitting first word' => array(
 				'<span>Plain text</span>',
-				1,
-				false,
-				true,
-				'...',
-			),
-			// Don't return invalid HTML
-			'Plain html over the limit splitting first word' => array(
```
Comment (mbabker) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: This one should have the key changed instead of deleted, looking at the array contents it looks like it's intended to be two separate test cases.

File: tests/unit/suites/libraries/cms/html/JHtmlStringTest.php [hunk 2]
```diff
@@ -173,27 +165,13 @@ public function getTestTruncateData()
 				false,
 				'Plain te...',
 			),
-			'HTML not allowed, no split' => array(
```
Comment (mbabker) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: Same as above.

File: tests/unit/suites/libraries/cms/html/JHtmlStringTest.php [hunk 3]
```diff
@@ -173,27 +165,13 @@ public function getTestTruncateData()
...
-				'...',
-			),
 			'First character is < with a maximum length of 1' => array(
 				'<div><span><i>Plain</i> <b>text</b> foo</span></div>',
 				1,
 				true,
 				false,
 				'...',
 			),
-			'HTML not allowed, no split' => array(
```
Comment (mbabker) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: Again.

File: tests/unit/suites/libraries/cms/html/JHtmlStringTest.php [hunk 4]
```diff
@@ -309,12 +287,6 @@ public function getTestTruncateComplexData()
 			),
 			'Plain html over the limit splitting first word' => array(
 				'<span>Plain text</span>',
-				4,
-				false,
-				'<span>P</span>...'
-			),
-			'Plain html over the limit splitting first word' => array(
```
Comment (mbabker) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: And one more 😉

File: tests/unit/suites/libraries/cms/html/JHtmlStringTest.php (hunk 1 above)
Comment (frankmayer) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: My thought was, they were probably forgotten to be removed... because the tests that I deleted, would never have run in the first place.

Comment (mbabker) [inline tests/unit/suites/libraries/cms/html/JHtmlStringTest.php]: That's true as well, but since the actual array contents differ (even though the keys are the same), to me it just looks like someone copied one element to create a new case but didn't change the key describing what the case was doing.

File: tests/unit/suites/libraries/joomla/access/JAccessTest.php [hunk 5]
```diff
@@ -65,7 +65,6 @@ public function casesCheck()
 		return array(
 			'valid_manager_admin_login' => array(44, 'core.login.admin', 1, true, 'Line:' . __LINE__ . ' Administrator group can login to admin'),
 			'valid_manager_login' => array(44, 'core.admin', 1, false, 'Line:' . __LINE__ . ' Administrator group cannot login to admin core'),
-			'super_user_admin' => array(42, 'core.admin', 3, true, 'Line:' . __LINE__ . ' Super User group can do anything'),
```
Comment (frankmayer) [inline tests/unit/suites/libraries/joomla/access/JAccessTest.php]: I haven't analyzed it further, yet. I'll check.

**Comment by frankmayer** [comment]
I have reverted some of the removals as those need a different PR and there is also a problem with a failing test (see https://github.com/joomla/joomla-cms/pull/13200#discussion_r92439293)

**Review by andrepereiradasilva — APPROVED** [review]

File: libraries/joomla/form/fields/password.php [hunk 6]
```diff
@@ -168,7 +168,6 @@ protected function getLayoutData()
 		// Initialize some field attributes.
 		$extraData = array(
 			'maxLength' => $this->maxLength,
-			'meter'     => $this->meter,
 			'threshold' => $this->threshold,
```
Comment (andrepereiradasilva) [inline libraries/joomla/form/fields/password.php]: Undo this change. The rest seems ok

Comment (frankmayer) [inline libraries/joomla/form/fields/password.php]: This must have been the last Github merge. I don't have this change in my branch...

Comment (frankmayer) [inline libraries/joomla/form/fields/password.php]: @andrepereiradasilva it was a double removal of the duplicate array 'meter' by two different commits (and users).

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on f30047e11c2a2d42e1e048edc21e0523a2e1ba06<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13200">issues.joomla.org/tracker/joomla-cms/13200</a>.</sub>

**Comment by frankmayer** [comment]
One more quick test (code review) please, so this minor PR can go RTC.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f30047e11c2a2d42e1e048edc21e0523a2e1ba06<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13200">issues.joomla.org/tracker/joomla-cms/13200</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.
